quora-service/src/main/java/com/upgrad/quora/service/business/AdminBusinessService.java

```diff
@@ -72,4 +74,30 @@ public UserEntity logoutUser(final String authorizationToken) throws SignOutRest
 
         return userAuthEntity.getUser();
     }
+
+    public void deleteUser(final String uuid, final String authorizationToken) throws UserNotFoundException, AuthorizationFailedException {
+
+        //UserEntity existingUser = userDao.getUserByUserName(userEntity.getUserName());
+
+        if (userDao.getUserById(uuid)==null) {
```

Comment:
1. We should have first validation for user authorization then for request details.

2. Code change:
```suggestion
           if (userDao.getUserByUuid(uuid) == null) {
```